agent/src/main/java/com/blackducksoftware/integration/hub/teamcity/agent/HubAgentBuildLogger.java

```diff
@@ -42,6 +44,17 @@ public void setLogLevel(final LogLevel level) {
 		loggerLevel = level;
 	}
 
+	public void setLogLevel(final String level) {
+		if (StringUtils.isNotBlank(level)) {
```

Comment:
any hope of putting this logic in to LogLevel with a hand-coded fromString method? if an appropriate LogLevel can't be found with whatever is passed in (be it null, blank, or whatever), just return LogLevel.INFO